pkg/openthread/contrib/netdev/openthread_netdev.c

```diff
@@ -94,10 +94,7 @@ static void *_openthread_event_loop(void *arg)
     /* init OpenThread */
     sInstance = otInstanceInitSingle();
 
-    /* enable OpenThread UART */
-    otPlatUartEnable();
```

Comment:
Maybe this should stay if not using the CLI (because the CLI enables UART internally)